src/styles/pagination.m.css

```diff
@@ -0,0 +1,34 @@
+@import './shared/variables.css';
+
+.navigation {
```

Comment:
Good point... right now it is calculated from the status text node's height (which is the same as the navigation node's height). Do you think it's important to account for scenarios where navigation may be taller than status?

Do you have a preferred approach for positioning this?